corehq/apps/translations/forms.py

```diff
@@ -213,58 +213,73 @@ def clean(self):
 
     @classmethod
     def form_for(cls, form_action):
-        if form_action == 'create':
-            return CreateAppTranslationsForm
-        elif form_action == 'update':
-            return UpdateAppTranslationsForm
+        if form_action == 'create_update':
+            return CreateUpdateAppTranslationsForm
         elif form_action == 'push':
             return PushAppTranslationsForm
         elif form_action == 'pull':
             return PullAppTranslationsForm
-        elif form_action == 'backup':
-            return BackUpAppTranslationsForm
         elif form_action == 'delete':
             return DeleteAppTranslationsForm
 
 
-class CreateAppTranslationsForm(AppTranslationsForm):
-    form_action = 'create'
+class CreateUpdateAppTranslationsForm(AppTranslationsForm):
```

Comment:
nit: `CreateOrUpdateAppTranslationsForm` or `ModifyAppTranslationsForm`?

Reply:
I had a long conversation with Graham about this haha. I felt there isn't really any word I feel like conveys both "Create" and "Update" equally (modifiy for instance doesn't really imply this can also create resources) so it would be the most straightforward to include both terms in the variable naming.

Reply:
Naming is probably one of the things that take the most time in programming. 

https://github.com/dimagi/commcare-hq/pull/34575/commits/f44568671239e18fd5e422682473aa3d36f39c2b
:+1: